coordinator/conf/config.json

```diff
@@ -7,17 +7,9 @@
     "batch_collection_time_sec": 180,
     "chunk_collection_time_sec": 180,
     "verifier": {
-      "mock_mode": true,
```

Comment:
mock mode is not used anymore?

Reply:
Yeah. iiuc this feature is never used in production, I guess that the original motivation is to bypass real verifier verification in some cases, while in all unit tests, "mock_verifier" tag is enabled, so coordinator uses mock verifier implementation so this path is never used.